MMM-COVID19.js

```diff
@@ -232,7 +236,11 @@ Module.register("MMM-COVID19", {
           statsDateCell = document.createElement("td");
 
       statsDateCell.innerHTML = this.translate('statistic taken at ') + this.countriesStats['statistic_taken_at'] + ' (UTC)'
-      statsDateCell.colSpan = "5";
+      if (this.config.delta) {
```

Comment:
this looks fine. I would keep only with this as the PR title says. The other thing (the sort thing is not related) could be done in another PR imho

Reply:
To simplify things, and since you're the owner of the code, it's certainly your call. But: the sorting works, I tested both ascending and descending by country_name or cases.

Reply:
@hweigel you are right about that. I was confussed about the way as it's being used the 2 configs together.